File: count.dart
void main() {
    print("Dart language");
  var startTime = DateTime.now().millisecondsSinceEpoch;
  var i = 0;
  while (i < 1000000000) {
    i++;
  }
  var endTime = DateTime.now().millisecondsSinceEpoch;
  var elapsedTime = (endTime - startTime) / 1000.0;
  print("time: $elapsedTime seconds");
}

File: count.lua
print("Lua")

local start_time = os.clock()

local i = 0
while i < 1000000000 do
    i = i + 1
end

local end_time = os.clock()
local elapsed_time = (end_time - start_time) * 1000  -- Convert to milliseconds

print("time:", elapsed_time, "milliseconds")
